STACK-UP-3/champs-frontend
# 172856134 Improving test coverage

#### Description
This chore improves test coverage.
#### Type of change

Please select the relevant option


#### How Has This Been Tested?


#### How to Test

Give instructions on how reviewers can verify your work
run `npm run test` test coverage should have been raised
#### Any background context you want to add (Optional)

Screenshots or GIFs

#### STORY ID

[Finishes #172856134]
